frontend/src/pages/LoginPage.tsx

```diff
@@ -1,21 +1,48 @@
-import React from 'react';
+import React, {FormEvent, useContext, useState} from 'react';
```

Comment:
```suggestion
import {FormEvent, useContext, useState} from 'react';
```

I don't know if React is used in the code below